crawl-ref/source/dat/descript/items.txt

```diff
@@ -95,11 +95,6 @@ battleaxe
 
 A large war axe with a formidable double-sided head.
 %%%%
-blowgun
-
-A long, light tube, open at both ends, through which various types of needles
-are propelled by puffs of breath at enemies. It makes very little noise.
-%%%%
```

Comment:
Does removing this description break anything when examining blowguns that still exist in a save game?

Reply:
No but it leaves an ugly error message where the description should be. Restored it.